freeipa/src/main/resources/schema/app/20200210124208_CB-4966_apply_ha_syncer_in_freeipa.sql

```diff
@@ -0,0 +1,217 @@
+-- // CB-4966 apply ha syncer in freeipa
+-- Migration SQL that makes the change goes here.
+
+DROP TABLE IF EXISTS QRTZ_FIRED_TRIGGERS;
+DROP TABLE IF EXISTS QRTZ_PAUSED_TRIGGER_GRPS;
+DROP TABLE IF EXISTS QRTZ_SCHEDULER_STATE;
+DROP TABLE IF EXISTS QRTZ_LOCKS;
+DROP TABLE IF EXISTS QRTZ_SIMPLE_TRIGGERS;
+DROP TABLE IF EXISTS QRTZ_CRON_TRIGGERS;
+DROP TABLE IF EXISTS QRTZ_SIMPROP_TRIGGERS;
+DROP TABLE IF EXISTS QRTZ_BLOB_TRIGGERS;
+DROP TABLE IF EXISTS QRTZ_TRIGGERS;
+DROP TABLE IF EXISTS QRTZ_JOB_DETAILS;
+DROP TABLE IF EXISTS QRTZ_CALENDARS;
```

Comment:
are these really necessary?

Reply:
it is template sql for quartz, I talked with @bbihari and he said, it is need to the HA scheduler.